Didi/busy indicator (#779)
telerik/maui-docs: knowledge-base/combobox-visual-states.md

---

> title: Set the Visual States of the ComboBox

Reviewer: ```suggestion
title: Setting the Visual States of the ComboBox
```

page_title: Configure the Visual States of the ComboBox Component - .NET MAUI Knowledge Base
description: Learn how to define the visual states of the Telerik UI for .NET MAUI ComboBox control.
type: how-to
slug: combobox-visual-states
tags: maui, combobox, visual states, .net maui combobox, focused, normal, disabled, hover, mouse over
res_type: kb
---

## Environment

| Version | Product | Author | 
| --- | --- | ---- | 
| 6.7.0 | ComboBox for .NET MAUI | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova)| 

## Description

How can I set the visual states of the Telerik UI for .NET MAUI ComboBox control?

## Solution

> To set the visual states of the ComboBox control, define the visual states, which is introduced by the `VisualStateManager` configuration. `VisualStateManager` defines a visual state group named `CommonStates` and supports the `Normal`, `Disabled`, `Focused`, and `MouseOver` values. It is recommended that you add configurations for all visual states that are available.

Reviewer: ```suggestion
To set the visual states of the ComboBox control, configure the `VisualStateManager`. The `VisualStateManager` defines a visual state group named `CommonStates` and supports the `Normal`, `Disabled`, `Focused`, and `MouseOver` values. As a general recommendation, always configure all available visual states.
```
